beacon_node/network/src/attestation_service/mod.rs

```diff
@@ -25,11 +25,8 @@ mod tests;
 
 /// The minimum number of slots ahead that we attempt to discover peers for a subscription. If the
 /// slot is less than this number, skip the peer discovery process.
-const MIN_PEER_DISCOVERY_SLOT_LOOK_AHEAD: u64 = 1;
-/// The number of slots ahead that we attempt to discover peers for a subscription. If the slot to
-/// attest to is greater than this, we queue a discovery request for this many slots prior to
-/// subscribing.
-const TARGET_PEER_DISCOVERY_SLOT_LOOK_AHEAD: u64 = 6;
+/// Subnet discovery query takes ~30secs, 3 slots take 36s.
+const MIN_PEER_DISCOVERY_SLOT_LOOK_AHEAD: u64 = 3;
```

Comment:
Might want to update the comment here and potentially change to 2. 
The reasons being:
- A query takes at most 30 seconds but can complete sooner. 
- We also might want to send a discovery request (even if it doesn't complete in time) because it will register any current peers associated with the required subnet as being necessary and prevent the peer manager from disconnecting them early.